From: Dmitry Eremin-Solenikov <dbaryshkov@gmail.com>
Date: Wed, 11 Sep 2019 23:40:57 +0300
Subject: src/engine.h: another qt5 fix

Signed-off-by: Dmitry Eremin-Solenikov <dbaryshkov@gmail.com>
---
 src/engine.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/engine.h b/src/engine.h
index f8d9327..4bddcb4 100644
--- a/src/engine.h
+++ b/src/engine.h
@@ -21,6 +21,7 @@
 #define __ENGINE_H__
 
 #include <QtCore/QByteArray>
+#include <QtCore/QObject>
 #include <QtCore/QStringList>
 #include <QtCore/QVector>
 
